samples/RewriteSample/Startup.cs

```diff
@@ -14,8 +14,8 @@ public class Startup
         public void Configure(IApplicationBuilder app, IHostingEnvironment env)
         {
             var options = new RewriteOptions()
-                .AddRedirect("(.*)/$", "$1")
-                .AddRewrite(@"app/(\d+)", "app?id=$1", skipRemainingRules: false)
+                .AddRedirect("^/(.*)/$", "$1")
+                .AddRewrite(@"^/app/(\d+)", "app?id=$1", skipRemainingRules: false)
```

Comment:
Won't this break the expected behavior of rules parsed from mod_rewrite and URLRewrite?